cleanlab/multilabel_classification/filter.py

```diff
@@ -0,0 +1,228 @@
+import warnings
+from typing import Optional, Union, Tuple, List, Any
+import numpy as np
+
+
+def find_label_issues(
+    labels: list,
+    pred_probs: np.ndarray,
+    return_indices_ranked_by: Optional[str] = None,
+    rank_by_kwargs={},
+    filter_by: str = "prune_by_noise_rate",
+    frac_noise: float = 1.0,
+    num_to_remove_per_class: Optional[int] = None,
+    min_examples_per_class=1,
+    confident_joint: Optional[np.ndarray] = None,
+    n_jobs: Optional[int] = None,
+    verbose: bool = False,
+) -> np.ndarray:
+    """
+    Finds label issues in multi-label classification data where each example can belong to more than one class.
+
+    Parameters
+    ----------
+    labels : List[List[int]]
+      List of noisy labels for multi-label classification where each example can belong to multiple classes (e.g. ``labels = [[1,2],[1],[0],[],...]`` indicates the first example in dataset belongs to both class 1 and class 2.
+
+
+    pred_probs : np.ndarray
+      An array of shape ``(N, K)`` of model-predicted probabilities,
+      ``P(label=k|x)``. Each row of this matrix corresponds
+      to an example `x` and contains the model-predicted probabilities that
+      `x` belongs to each possible class, for each of the K classes. The
+      columns must be ordered such that these probabilities correspond to
+      class 0, 1, ..., K-1. They need not sum to 1.0
+
+
+    return_indices_ranked_by : {None, 'self_confidence', 'normalized_margin', 'confidence_weighted_entropy'}, default=None
+      Refer to documentation for this argument in :py:func:`filter.find_label_issues <cleanlab.filter.find_label_issues>` for details.
```

Comment:
please copy the first sentence of the docstring for each arg that at least quickly summarizes what the arg is, before you refer user to the linked docstring. Do this for every arg here.

Reply:
Added here: https://github.com/cleanlab/cleanlab/blob/25672dc504471dd2d450642942555e44528565dc/cleanlab/multilabel_classification/filter.py#L43